src/models/queries/viewMenFashion.js

```diff
@@ -0,0 +1,14 @@
+const dbconnection = require('../database/db_connection');
+
+const viewMenFashion = (cb) => {
+  const viewMenFashionSql = {
+    text: 'select * from cloths where gender = \'m\'',
+  };
+  dbconnection.query(viewMenFashionSql, (dataBaseConnectionError, menFashion) => {
+    if (dataBaseConnectionError) return cb(dataBaseConnectionError);
+
+    return cb(null, menFashion.rows);
```

Comment:
i don't know why you need to return cb(...) ! 
i guess the callback return self auto ! 
Is there any error happens with you when delete return ?

Reply:
it's because the function needs return at the end!